index.html

```diff
@@ -178,10 +251,10 @@ <h2>Contact me</h2>
         <form action="https://formspree.io/f/mwkzpvlw" id="form" method="post">
```

Comment:
- [ ] [OPTIONAL] Good job decreasing the form fields width size to make it match the Design, however, the form is not aligned properly. kindly align it so it is in the center of the page like in the Figma design

Reply:
I have done that. Thank you